sktm/executable.py

```diff
@@ -13,11 +13,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
-
-import argparse
 import ConfigParser
+import argparse
 import logging
 import os
+
```

Comment:
I think this should be in some other commit.

Reply:
I'm not sure what you would like changed here.

Reply:
Ah, sorry I wasn't clear. I mean it seems to me this is just rearranging the order of `import`s and fixing formatting and is not part of "Make sktm installable via pip" change. Perhaps this should go into the "Fix flake8 issues" commit?